src/main/java/graphql/nadel/TypeDefinitionsWithRuntimeWiring.java

```diff
@@ -0,0 +1,56 @@
+package graphql.nadel;
+
+import graphql.schema.idl.TypeDefinitionRegistry;
+import graphql.schema.idl.TypeRuntimeWiring;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+
+/**
+ * Stores {@link TypeDefinitionRegistry} and collection of {@link TypeRuntimeWiring}s.
+ */
+public class TypeDefinitionsWithRuntimeWiring {
+    private final TypeDefinitionRegistry typeDefinitionRegistry;
+    private final List<TypeRuntimeWiring> typeRuntimeWirings;
```

Comment:
why using this List of TypeRuntimeWiring here? 
Braid expects `Consumer<RuntimeWiring.Builder>` which I think we should also use ....

Reply:
I agree that it is nice not to have Consumer but just data,
but the `RuntimeWiring` is not just `TypeRuntimeWiring`: it is also scalars, directivesWiring and more.
By restricting it to `TypeRuntimeWiring` you are restricting the whole wiring from the outside approach: You need `RuntimeWiring` to be able to do everything.

The reason why it is actually a `Consumer<Builder>` and not just `RuntimeWiring` is a braid/ graphql-java shortcoming, because Braid wants to combine different `RuntimeWiring` and there is no easy way to combine them atm, so they give you the Builder.

Reply:
understood. Fixed to use Consumer<builder> as well